packages/@aws-cdk/alexa-ask/package.json

```diff
@@ -61,12 +61,14 @@
     "pkglint": "^0.22.0"
   },
   "dependencies": {
-    "@aws-cdk/cdk": "^0.22.0"
+    "@aws-cdk/cdk": "^0.22.0",
+    "@aws-cdk/aws-codepipeline-api": "^0.22.0"
```

Comment:
This should be above the `cdk` dependency (we try to keep them in alphabetical order).